haas/api.py

```diff
@@ -133,7 +133,60 @@ def project_detach_node(project, node):
     project.nodes.remove(node)
     db.session.commit()
 
+@rest_call('PUT', '/network/<network>/access/<project>')
+def network_grant_project_access(project, network):
+   """Add access to <network> to <project>.
```

Comment:
Per PEP257, you should have a blank line after the one-line summary. Same goes for the other docstrings.